### PR TITLE
chore: Bigtable new sample format for write and delete samples

### PR DESCRIPTION
**Pull request by saranshdhingra**

### CODE REVIEW

File: bigtable/test/writeTest.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public static function tearDownAfterClass(): void
 
     public function testWriteSimple()
     {
-        $output = $this->runSnippet('writes/write_simple', [
+        $output = $this->runSnippet('write_simple', [
```
Comment (bshaffer) [inline bigtable/test/writeTest.php:49]: All of these should be changed to `runFunctionSnippet`
```suggestion
        $output = $this->runFunctionSnippet('write_simple', [
```

File: bigtable/src/delete_cluster.php [hunk 2]
```diff
@@ -22,39 +22,38 @@
...
+/**
+ * Delete a cluster
+ * @param string $projectId The Google Cloud project ID
+ * @param string $instanceId The ID of the Bigtable instance
+ * @param string $clusterId The ID of the cluster to be deleted
+ */
+function delete_cluster(
+    string $projectId,
+    string $instanceId,
+    string  $clusterId
```
Comment (bshaffer) [inline bigtable/src/delete_cluster.php:38]:
```suggestion
    string $clusterId
```

Comment (saranshdhingra) [reply bigtable/src/delete_cluster.php:38]: Somehow `php-cs-fixer` didn't fix this 🤔 . Good catch!

File: bigtable/test/BigtableTestTrait.php [hunk 3]
```diff
@@ -96,7 +96,7 @@ public static function deleteBigtableInstance()
         self::$instanceAdminClient->deleteInstance($instanceName);
     }
 
-    private static function runSnippet($sampleName, $params = [])
+    private static function runFunctionSnippet($sampleName, $params = [])
```
Comment (bshaffer) [inline bigtable/test/BigtableTestTrait.php:99]: There is a function `runFunctionSnippet` built into `TestTrait` which should be handling this.
```suggestion
    private static function runSnippet($sampleName, $params = [])
```

**Review by bshaffer — APPROVED** [review]

**Review by bshaffer — APPROVED** [review]

**Pull request merged**